src/precompiles/ArbNativeTokenManager.sol

```diff
@@ -0,0 +1,28 @@
+// Copyright 2021-2025, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro-contracts/blob/main/LICENSE.md
+// SPDX-License-Identifier: BUSL-1.1
+
+pragma solidity >=0.4.21 <0.9.0;
+
+/**
+ * @title Enables minting and burning native tokens.
+ * @notice Authorized callers are added/removed through ArbOwner precompile.
+ * Available in ArbOS version 41
```

Comment:
what address will this redeployed at?

Reply:
In this PR https://github.com/OffchainLabs/go-ethereum/pull/447/files#diff-97fb0269a5d0cee29ffb8df43b3c0527702d5a90eedc8e06d7cab46795164391R25

We suggest `0x73` does that work for you?